Junit4/src/test/java/com/knoldus/StringUtilsTest.java

```diff
@@ -0,0 +1,74 @@
+package com.knoldus;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class StringUtilsTest {
+    StringUtils stringUtils;
+
+    @Before
+    public void setup(){
+        stringUtils = new StringUtils();
+    }
+
+    // reverseStringTestMethod method to test the reverse method on given string
+    @Test
+    public void reverseStringTestMethodForNonEmptyString(){
```

Comment:
testReverseString_givenExpectedInput -- like this, we need to have the name that reflects scenario as suffix.